docker_services_cli/config.py

```diff
@@ -111,7 +113,7 @@
     "postgresql": POSTGRESQL,
     "mysql": MYSQL,
     "redis": REDIS,
-    "memcached": MEMCACHED,
+    # "memcached": MEMCACHED,
```

Comment:
doubt: is this meant to be commented out? I see Memcached things are comment. Just wondering: If there is no support, should it just be removed?

Reply:
Agreed, I will remove since there is no support.